### DISCUSSION TITLE
Dropdown type-ahead:

### DISCUSSION THREAD

**Discussion by metaleap** [carbon-design-system/carbon-components-svelte#1974] [Q&A] [answered]
When the component is focused, dropdown type-ahead works in the [React live demo](https://carbondesignsystem.com/components/dropdown/code) but not in the [Svelte one](https://svelte.carbondesignsystem.com/framed/Dropdown/DropdownSlot?theme=white).

I'm curious, how come, how does this happen? What's the level of reuse vs re-implementation among the React / Vue / Svelte / Angular versions, do they all try to catch up to each other, which one is the "canonical" / source-of-truth / "native" one? Are they differently prioritized / endowed within IBM? Or did I stumble on a rare simple plain-old lil' bug :grin:

Chromium 125.0.6422.60 on Linux.

**Comment by theetrain** [comment, marked as answer]
Hello, I created an issue from your report https://github.com/carbon-design-system/carbon-components-svelte/issues/1975

>What's the level of reuse vs re-implementation among the React / Vue / Svelte / Angular versions, do they all try to catch up to each other, which one is the "canonical" / source-of-truth / "native" one? Are they differently prioritized / endowed within IBM?

`carbon-components-svelte` is a community-maintained port of `@carbon/react`. It's meant to match the *user experience* of `@carbon/react` while catering to the niceties of Svelte development.

More context can be found in: https://github.com/carbon-design-system/carbon-components-svelte/discussions/1614.

**Comment by metonym** [comment]
Released in [v0.93.0](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.93.0)
